oauth2/oauth2-resource-server/src/test/java/org/springframework/security/oauth2/server/resource/introspection/NimbusReactiveOpaqueTokenIntrospectorTests.java

```diff
@@ -244,7 +244,7 @@ private WebClient mockResponse(String response) {
 		given(clientResponse.statusCode()).willReturn(HttpStatus.OK);
 		given(clientResponse.bodyToMono(String.class)).willReturn(Mono.just(response));
 		ClientResponse.Headers headers = mock(ClientResponse.Headers.class);
-		given(headers.contentType()).willReturn(Optional.of(MediaType.APPLICATION_JSON_UTF8));
+		given(headers.contentType()).willReturn(Optional.of(MediaType.APPLICATION_JSON));
```

Comment:
Should there be a similar test on the servlet side?

Reply:
The tests for NimbusOpaqueTokenIntrospector already use the `MediaType.APPLICATION_JSON` instead of the deprecated one so I guess there is no need to change them.